pipeline/src/ui.provider.pact.test.ts

```diff
@@ -134,20 +149,41 @@ async function setupEmptyState(): Promise<void> {
   return Promise.resolve();
 }
 
-async function setupSomePipelineConfigs(): Promise<void> {
+async function setupSomePipelineConfigsWithoutSchemas(): Promise<void> {
+  await setupSomePipelineConfigs(false);
+}
+
+async function setupSomePipelineConfigsWithSchemas(): Promise<void> {
+  await setupSomePipelineConfigs(true);
+}
+
+async function setupSomePipelineConfigs(withSchemas: boolean): Promise<void> {
   clearState();
-  addSamplePipelineConfig(++nextPipelineConfigId, 2, true);
-  addSamplePipelineConfig(++nextPipelineConfigId, 3, false);
-  addSamplePipelineConfig(++nextPipelineConfigId, 2, false);
+  addSamplePipelineConfig(2, withSchemas);
+  addSamplePipelineConfig(3, withSchemas);
+  addSamplePipelineConfig(2, withSchemas);
 
   return Promise.resolve();
 }
 
-async function setupSomePipelineTransformedData(): Promise<void> {
+async function setupSomePipelineTransformedDataWithHealthStatusOk(): Promise<void> {
```

Comment:
I think instead of adding another method, you could provide in the state something like `() => setupSomePipelineTransformedData(HealthStatus.OK)`